packages/flutter/test/material/scaffold_test.dart

```diff
@@ -389,6 +389,16 @@ void main() {
     expect(scrollable.position.pixels, equals(0.0));
   }, variant: const TargetPlatformVariant(<TargetPlatform>{ TargetPlatform.iOS,  TargetPlatform.macOS }));
 
+  testWidgets('Tapping the status bar scrolls to top use more appropriate curve', (WidgetTester tester) async {
```

Comment:
Is this testing anything different from the previous `Tapping the status bar scrolls to top` test?
@goderbauer any hints for how to test an animation curve like this? Seems tricky.

Reply:
It's tricky. Instead of running through the entire animation at once with `pumpAndSettle`, you could step through the animation with `pump(duration)` and assert that at key moments of the animation things are were you expect them to be based on the curve.